src/fairseq2/setup/_po_finetune_units.py

```diff
@@ -85,7 +87,11 @@ def register_online_finetune_units(context: RuntimeContext) -> None:
     # MathVerify
     handler = MathVerifyHandler()
     registry.register(handler.name, handler)
-    
+
     # GenerativePointwiseVerifier
     handler = GenerativePointwiseVerifierHandler()
     registry.register(handler.name, handler)
+
+    # NoEnvAtheneRewardPipeline
+    handler = NoEnvAtheneRewardPipeline
```

Comment:
should this not be in the `VLLMOutputRewardHandler` registry?

Reply:
> should this not be in the `VLLMOutputRewardHandler` registry?

yes, ideally this should be different registry for e.g. remote actor classes